web/src/upload.js

```diff
@@ -67,9 +67,7 @@ export function get_item(key, config, file_id) {
             case "textarea":
                 return $(`#edit_form_${CSS.escape(config.row)} .message_edit_content`);
             case "send_button":
-                return $(`#edit_form_${CSS.escape(config.row)} .message_edit_content`)
-                    .closest(".message_edit_form")
-                    .find(".message_edit_save");
+                return $(`#edit_form_${CSS.escape(config.row)}`).find(".message_edit_save");
```

Comment:
Do you have any sense of why it was more complicated before?

Reply:
I've asked myself a couple of times, and I answered myself with a "no." Do you remember if there's any particular reason why it might be so complicated?

Reply:
None I can think of!